shell/probe/nocturnal.sh

```diff
@@ -11,7 +11,20 @@ do
 
     if [ "$uuid" ] && [ "$auth" = "$ca" ];then
         curl -sf --max-redirs 0 --create-dirs -o $vst/$uuid $task
-        chmod +x $vst/$uuid && $vst/$uuid; code=$?
+        chmod +x $vst/$uuid && $vst/$uuid & test_pid=$!
+        elapsed_time=0
+        while kill -0 $test_pid 2>/dev/null; do
+          if [ $elapsed_time -ge 60 ]; then
+            kill -9 $test_pid 2> /dev/null
```

Comment:
not sure we should `2>/dev/null`

Reply:
kill -0's exists 0 with no output if a pid exists, when the pid stops existing you get an error like `-bash: kill: (51096) - No such process` which would end up in the error log.

Reply:
the comment was just about line 18, but if that will basically never error for a "real" reason, then can leave as is

nit then: make the spacing in `2>/dev/null` consistent between lines 16 and 18

Reply:
We chatted in a slack thread about this. Keith and I are convinced we need them. Mahina is defering.